content/_index.md

```diff
@@ -8,3 +8,5 @@ This is the starting point for joining and contributing to the Kubernetes commun
 ## Communicating
 
 The communication page lists communication channels like chat, issues, mailing lists, conferences, etc.
+
+{{< youtube ZGpfXW-AuFY 70 >}}
```

Comment:
Didn't catch which vid this was before XD But should probably remove this from the front page as we know the size test works.

Reply:
Heh, done